README.md

```diff
@@ -5,22 +5,26 @@ OpenVPN for Android
 
 Description
 ------------
-With the new VPNService of Android API level 14+ (Ice Cream Sandwich) it is possible to create a VPN service that does not need root access. This project is a port of OpenVPN.
+In Android 4.0 (Ice Cream Sandwich) a VPN service can be made without root access, using the VPNService of API level 14+. \
+This project is a port of OpenVPN.
 
 <a href="https://f-droid.org/repository/browse/?fdid=de.blinkt.openvpn" target="_blank">
 <img src="https://f-droid.org/badge/get-it-on.png" alt="Get it on F-Droid" height="80"/></a>
 <a href="https://play.google.com/store/apps/details?id=de.blinkt.openvpn" target="_blank">
 <img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" alt="Get it on Google Play" height="80"/></a>
 
 Developing
----------------
-If you want to develop on ics-openvpn please read the [doc/README.txt](https://github.com/schwabe/ics-openvpn/blob/master/doc/README.txt) *before* opening issues or emailing me. 
+----------
+Read the [doc/README.txt](https://github.com/schwabe/ics-openvpn/blob/master/doc/README.txt) *before* opening issues or e-mailing. 
 
-Also please note that before contributing to the project that I would like to retain my ability to relicense the project for different third parties and therefore probably need a contributer's agreement from any contributing party. To get started, [sign the Contributor License Agreement](https://www.clahub.com/agreements/schwabe/ics-openvpn).
+To retain my ability to re-license the project for different third-parties I probably need a [contributor agreement](https://www.clahub.com/agreements/schwabe/ics-openvpn) from any contributing party.
 
-You can help
+Translations
 ------------
-Even if you are no programmer you can help by translating the OpenVPN client into your native language. [Crowdin provides a free service for non commercial open source projects](https://crowdin.net/project/ics-openvpn/invite) (Fixing/completing existing translations is very welcome as well)
+Help [translate](https://hosted.weblate.org/projects/openvpn-for-android/) the app to your language.
+<a href="https://hosted.weblate.org/engage/openvpn-for-android/">
+<img src="https://hosted.weblate.org/widgets/openvpn-for-android/-/horizontal-blue.svg" alt="Oversettelsesstatus" />
+</a>
```

Comment:
You just assumed that I will switch to weblate. I haven't decided that yet. I haven't found the time to actually look into weblate and if it is really that much better as you claim, so don't make decisions for me.